ui/src/App.tsx

```diff
@@ -214,6 +215,12 @@ const App = () => {
                 selectable={true}
                 eventClick={openModal}
                 height="100vh"
+                eventDataTransform={(event) => {
+                  if (event.allDay) {
+                    event.end = moment(event.end).add(1, "days").toDate();
+                  }
+                  return event;
+                }}
```

Comment:
We could pull this logic out into its own function and then add unit tests for it.